### ISSUE TITLE
question about training paradigm

### ISSUE THREAD

**Issue by zhuang-li** [sail-sg/sdft#5]
Hi, this is a very interesting work! One thing I don't understand is whether the self-distillation is rewriting using Llama2-chat and further fine-tuning Llama2-chat as well, or is it just fine-tuning Llama2?

**Comment by rickyang1114** [comment]
Hello, thanks for your interest!&nbsp;


We actually utilize Llama-2-chat throughout the whole process.&nbsp;



---Original---
From: "Zhuang ***@***.***&gt;
Date: Sat, Jun 29, 2024 17:49 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [sail-sg/sdft] question about training paradigm (Issue #5)




 
Hi, this is a very interesting work! One thing I don't understand is whether the self-distillation is rewriting using Llama2-chat and further fine-tuning Llama2-chat as well, or is it just fine-tuning Llama2?

**Comment by zhuang-li** [comment]
Thank you for your prompt response! Interestingly, we have a concurrent project that closely aligns with the rewriting technique you described. Although rewriting isn't our work's primary focus, we just applied this method to train our base model for some empirical study. But we did discover findings similar to yours. We've only recently become aware of your excellent prior work in this area! Our research explores the technique's application in a different context and offers additional analysis of its effectiveness. Hope it can help with your research as well:

https://arxiv.org/pdf/2406.10882
